### ISSUE TITLE
Rex::Service::SuSE on openSUSE Leap

### ISSUE THREAD

**Issue by LeMerP** [RexOps/Rex#1114]
Since openSUSE Leap uses systemd instead of System V as service manager, a service does not need to be symlinked under /etc/rc.d anymore.
Running '_service "foo" => "reload";_' on such machine will often produce strange output like:
```
Rex::Helper::Run::i_run('/etc/rc.d/bareos-fd restart', 'nohup', 1) called at /usr/lib/perl5/site_perl/5.18.2/Rex/Service/Base.pm line 58
	Rex::Service::Base::_execute('Rex::Service::SuSE=HASH(0x3d25488)', '/etc/rc.d/bareos-fd restart') called at /usr/lib/perl5/site_perl/5.18.2/Rex/Service/Base.pm line 92
```
I would suggest to use the _/usr/sbin/service %s_ command for operations like "start, restart, stop, reload, status" and _/usr/bin/systemctl_ to manage service autostart.

**Comment by LeMerP** [comment]
This issue is related to the development-1.x branch. master branch is working as expected.

**Comment by ferki** [comment]
> This issue is related to the development-1.x branch

So I'd like to keep it open to don't accidentally release this bug with the next 1.x release :)

Thanks for reporting!

**Comment by LeMerP** [comment]
As I can see, there are systemd modules (Rex::Service::SuSE::systemd). It seems they just will not be used.

**Comment by ferki** [comment]
I think what we need to do:
- teach rex to differentiate between opensuse and opensuse leap
- and based on that, use systemd or not for service management by default

Until then [`service_provider_for`](http://www.rexify.org/docs/api/1.4/rex/commands/service.pm.html) can be used to override the automatic selection:
```
service_provider_for SuSE => 'systemd';
```

**Comment by krimdomu** [comment]
This might be a problem of the development-1.x branch solved by #1126 
i have to create a suse leap test system

**Comment by krimdomu** [comment]
i just tested it, and indeed it was fixed with #1126
